modules/user-provided-certificates-for-api-server.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// *authentication/certificate-types-descriptions.adoc
+
+[id="user-provided-certificates-for-the-api-server_{context}"]
+= User-provided certificates for the API server
+
+[discrete]
+== Purpose
+
+The API server is accessible by clients external to the cluster at
+`api.<cluster_name>.<base_domain>`. You might want clients to access the API
+server at a different host name or without the need to distribute the
+cluster-managed certificate authority (CA) certificates to the clients. The
+administrator must set a custom default certificate to be used by the API server
+when serving content.
+
+[discrete]
+== Location
+
+The user-provided certificates must be provided in a `kubernetes.io/tls` type
+`Secret` in the `openshift-ingress` namespace. Update the API server cluster
```

Comment:
@ahardin-rh I remembered https://github.com/openshift/openshift-docs/pull/18254#discussion_r384732697 requested change for  `openshift-ingress` to `openshift-config`. Not sure why still seeing `openshift-ingress` :)

Reply:
@ahardin-rh , didn't see the change in your new update. Here please change `openshift-ingress` to `openshift-config`, thanks

Reply:
Sorry about that! I thought I saved the change, but I guess not. It's updated now!